### ISSUE TITLE
stack build fails: "aeson-0.9.0.1 must match >=0.11 (latest applicable is 1.2.2.0)"

### ISSUE THREAD

**Issue by Qwertie-** [SFTtech/openage-masterserver#7]
When running `stack build` in the repo I get the output:

```
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for openage-masterserver-0.1.0.0:
    aeson-0.9.0.1 must match >=0.11 (latest applicable is 1.2.2.0)

Plan construction failed.
```

**Comment by TheJJ** [comment]
Err, why `aeson-0.9.0.1`? Isn't the latest applicable good enough for the version requirement?

**Comment by Qwertie-** [comment]
@TheJJ Don't know, that's just what stack chose.

**Comment by TheJJ** [comment]
Fixed by #8
